traffic_monitor/README.md

```diff
@@ -16,5 +16,10 @@
     specific language governing permissions and limitations
     under the License.
 -->
+# Traffic Monitor
 
-To run: `./build.sh && traffic_monitor --opsCfg ./conf/traffic_ops.cfg -config ./conf/traffic_monitor.cfg`
+* To run:
+  * `./build.sh && traffic_monitor --opsCfg ./conf/traffic_ops.cfg -config ./conf/traffic_monitor.cfg`
```

Comment:
This marks this as a sub-list, renders like:

```html
<ul>
  <li>To run:</li>
  <ul>
    <li><code>./build.sh && traffic_monitor --opsCfg ./conf/traffic_ops.cfg -config ./conf/traffic_monitor.cfg</code></li>
  </ul>
</ul>
```
Which isn't strictly accurate

Reply:
oops, this should've been part of the review :P

Reply:
Thanks for pointing that out! I'm not a markdown expert, good to know. I think I got the formatting right this time.